src/commands/dev/dev.ts

```diff
@@ -137,6 +137,11 @@ export const dev = async (options: OptionValues, command: BaseCommand) => {
   try {
     settings = await detectServerSettings(devConfig, options, command)
 
+    if (process.env.NETLIFY_INCLUDE_DEV_SERVER_PLUGIN) {
+      log(`${NETLIFYDEVLOG} Including dev server plugin`)
```

Comment:
Is this the customer-facing name we're going with? And is there value in showing this message? I believe we'll still show a generic message from Netlify Build indicating that this plugin is being loaded? I would consider printing this only if `debug` is true.